specs/cypress/open-cypress-mobile-viewport.yaml

```diff
@@ -0,0 +1,15 @@
+name: Open Cypress With A Mobile Viewport
```

Comment:
uber nit: can we make "A" lowercase here ("a"). Same with the Run cypress workflow as well

Reply:
[cb5ec62](https://github.com/warpdotdev/workflows/pull/46/commits/cb5ec62817f7e865bcf75f33b54c8791d0d5b7de) resolves this